CHANGELOG.md

```diff
@@ -109,6 +109,25 @@
   the Erlang target if any of their arguments were discarded.
   ([Giacomo Cavalieri](https://github.com/giacomocavalieri))
 
+- The compiler now narrows the variant of custom types when pattern matching,
+  allowing for safe record updates if the variant is known:
```

Comment:
Talking with Louis and Hayleigh at code beam we agreed that we should be really clear in framing this as "safe record updates" instead of _narrowing_ of any type to avoid misleading people into thinking this is type narrowing. This could be reworded as:
```suggestion
- The compiler can now allow you to perform record updates for specific
  variants of a type after pattern matching on it:
```

Reply:
Sounds good!